ocdskingfisherprocess/database.py

```diff
@@ -620,41 +620,31 @@ def get_releases_to_check(self, collection_id, override_schema_version=None):
             sql += """ LEFT JOIN release_check ON release_check.release_id = release_with_collection.id
                           AND release_check.override_schema_version = :override_schema_version
                        LEFT JOIN release_check_error ON release_check_error.release_id = release_check_error.id
-                          AND release_check_error.override_schema_version = :override_schema_version """
+                          AND release_check_error.override_schema_version = :override_schema_version
+                       LEFT JOIN package_data on package_data.id = package_data_id
+                       WHERE release_with_collection.collection_id = :collection_id
+                       AND release_check.id IS NULL AND release_check_error.id IS NULL
+                       AND coalesce(data ->> 'version', '1.0') <> :override_schema_version"""
```

Comment:
@kindly This seems to be saying "Check only data that isn't 1.1". But "override_schema_version" means "Force OCDS 1.1 checks to be run on OCDS 1.0 data (instead of OCDS 1.0 checks)". Checks should still be run on OCDS 1.1 data...

Reply:
I thought there were essentially two check runs.  One on the original versions which will run checks on the 1.1 data with 1.1 schema (and 1.0 with 1.0 schema). Then there will be a second optional run, with `override_schema_version` = 1.1 which ignores the 1.1 data as those checks have already been run and only checks non 1.1 data.

Reply:
Aha, thanks, I hadn't realized it did two runs. I would have expected just one run.